docs/Admin.md

```diff
@@ -184,6 +184,21 @@ await admin.setOffsets({
 })
 ```
 
+## <a name="describe-cluster"></a> Describe cluster
+
+Allows you to get information about the broker cluster.
+
+```javascript
+await admin.describeCluster()
+// {
+//   brokers: [
+//     { nodeId: 0, host: 'localhost', port: 9092, connected: true }
+//   ],
+//   controller: 0,
+//   clusterId: 'f8QmWTB8SQSLE6C99G4qzA'
+// }
+```
+
```

Comment:
Straightforward example, but maybe missing a little bit of context of situations _why_ you might want to do this. I always find example use cases very useful when I'm new to a library when trying to decide what I _have_ to do or _should_ be doing, what best practices are, etc.

Reply:
I agree that a use-case would be nice. Can you think of any beyond the current one? The current use-case we have is to show a list of brokers and their current connection status in a vscode plugin, which is a bit specific. 😄 I tried finding a KIP for when `DescribeCluster` was introduced, but I couldn't find one, so maybe it was added before they were doing KIPs.

Reply:
I think the current one is probably the valid use of it. Even if that's a bit on the nose, it would be a good to indication that this is mostly useful for monitoring / ops and _not_ essential to control flow or writing consumers. To a consumer, the shape of a cluster shouldn't really matter, instead just be focused on topics, partitions, offsets, groups, etc.

Reply:
Fixed in daae6ed